### PR TITLE
This makes the feature of custom background , login specific.

### LINKED ISSUE #199: To make certain features login specific
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/susi_server

**Current behavior:**
All the existing features are available to every user.

**Expected behavior:**
To make certain features only login specific that provides extra perks to logged in user

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by stealthanthrax**
Fixes #199 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Since all the existing features are available to every user, therefore, there is no motive for a user to log in.Hence certain features are made login specific;

#### Changes proposed in this pull request:

To make the custom background feature as login specific and also make the upcoming features, login specific

### CODE REVIEW

**Comment by stealthanthrax** [comment]
@zamhaq , have a look

File: src/js/allowmic.js [hunk 1]
```diff
@@ -2,6 +2,21 @@ var micAccess = document.getElementById("micAccess");
 var backgroundChange = document.getElementById("backgroundChange");
 var backUrl = document.getElementById("backUrl");
 var theValue;
+var warning = document.getElementById("warning");
+var customBackground = document.getElementById("customBackground");
+var checkLogin = localStorage.getItem("checkLogin");
+
+window.onload = () => {
+    console.log("The login status is " + checkLogin);
+    if(checkLogin === "true") {
```
Comment (zamhaq) [inline src/js/allowmic.js]: Also, add `and` condition for checking if `accessToken` is present. Currently, the access token is not being saved in the in the `chrome.storage`.

Comment (zamhaq) [inline src/js/allowmic.js]: Also, check if `accessToken` is present. Currently, the `accessToken` is not being saved. I think it is misspelled, when I last checked while working on #165

File: src/js/allowmic.js [hunk 2]
```diff
@@ -2,6 +2,21 @@ var micAccess = document.getElementById("micAccess");
...
+        customBackground.style.display = "block";
+    }
+    else {
+        warning.innerHTML = "Please login for some usefull features";
```
Comment (zamhaq) [inline src/js/allowmic.js]: small typo: `useful`.

**Comment by stealthanthrax** [comment]
@zamhaq ,have a look now

**Comment by stealthanthrax** [comment]
@zamhaq ,please have a look now

**Comment by stealthanthrax** [comment]
done @gabru-md

**Comment by stealthanthrax** [comment]
before login \/
screen shot 2017-12-30 at 12 12 46 am
after login \/
screen shot 2017-12-30 at 12 18 56 am

**Comment by stealthanthrax** [comment]
@ms10398 @zamhaq , have a look

**Comment by stealthanthrax** [comment]
@akshatnitd ,since on the webchat the custom background is a login specific feature and also not making certain features as login specific will not encourage the user to login.

**Comment by ms10398** [comment]
And yes @akshatnitd as @stealthanthrax said we are following the way its implemented in web chat

**Comment by akshatnitd** [comment]
@ms10398 @ms10398 Okay cool 👍

**Comment by mariobehling** [comment]
Merging. Thanks!

**Pull request merged**